docs/how-it-works.md

```diff
@@ -22,23 +22,28 @@ The `.api/` directory is where the CLI installation process stores all of its in
 
 ```
 .api/
-├── api.json             // The `package-lock.json` equivalent that records
-|                        // everything that's installed, when it was installed,
-|                        // what the original source was, and what version of
-|                        // `api` was used.
+├── api.json               // The `package-lock.json` equivalent that records
+|                          // everything that's installed, when it was installed,
+|                          // what language was generated, what the original
+|                          // source was, and what version of `api` was used.
```

Comment:
just b/c i don't love the idea of us having to update these docs every time we update the `api.json` schema

```suggestion
|                          // metadata about the installed SDKs.
```